web/src/components/nomination/PronounInput.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+  <v-col
+    cols="12"
+    md="6"
+    xs="12"
+  >
+    <v-combobox
+      ref="pronouns"
+      :label="$t('nominateSpeaker.pronouns.label')"
+      :hint="$t('nominateSpeaker.pronouns.hint')"
+      :items="pronouns"
+      persistent-hint
+      outlined
+      :value="value"
+      @input="$emit('input', $event)"
+    />
+  </v-col>
+</template>
+
+<script>
+export default {
+  props: {
+    value: {
+      type: String,
+      required: true,
+    },
+  },
+  data() {
+    return {
+      pronouns: [],
+      error: null,
+    };
+  },
+  mounted() {
+    this.$getPronouns(this.setPronouns, this.setError);
```

Comment:
Since we offer only 2 suggestions and allow anything else to be entered, do we really need to fetch all options from Airtable? Maybe just hardcoding the 2 options and allowing free text entry is enough here (and saves us this extra query).

Reply:
I thought this would give us the flexibility to change things if certain pronouns become more popular, but could also hardcode it.